scripts/fast-reboot

```diff
@@ -738,25 +738,12 @@ for service in ${SERVICES_TO_STOP}; do
     fi
 done
 
-# Kill other containers to make the reboot faster
-# We call `docker kill ...` to ensure the container stops as quickly as possible,
-# then immediately call `systemctl stop ...` to prevent the service from
-# restarting the container automatically.
-debug "Stopping all remaining containers ..."
 if test -f /usr/local/bin/ctrmgr_tools.py
 then
+    debug "Stopping all remaining containers ..."
```

Comment:
Is this debug flag misplaced, considering the kill/shutdown happens implicitly now through systemd?